test/lib/ansible_test/_internal/commands/integration/cloud/galaxy.env

```diff
@@ -0,0 +1,33 @@
+PULP_CONTENT_ORIGIN=http://galaxy-pulp:80
+PULP_ANSIBLE_API_HOSTNAME=http://galaxy-pulp:80
+PULP_GALAXY_API_PATH_PREFIX=/api/galaxy/
+# These paths are unique to the container image which has an nginx location for /pulp/content to route requests to the content backend
+PULP_ANSIBLE_CONTENT_HOSTNAME=http://galaxy-pulp:80/pulp/content/api/galaxy/v3/artifacts/collections/
+PULP_CONTENT_PATH_PREFIX=/pulp/content/api/galaxy/v3/artifacts/collections/
+
+PULP_GALAXY_AUTHENTICATION_CLASSES=['rest_framework.authentication.SessionAuthentication', 'rest_framework.authentication.TokenAuthentication', 'rest_framework.authentication.BasicAuthentication', 'django.contrib.auth.backends.ModelBackend']
+
+# This should probably be false see https://issues.redhat.com/browse/AAH-2328
+PULP_GALAXY_REQUIRE_CONTENT_APPROVAL=true
+PULP_GALAXY_DEPLOYMENT_MODE=standalone
+PULP_GALAXY_AUTO_SIGN_COLLECTIONS=false
```

Comment:
Wouldn't AH actual be auto signing collections?

Reply:
Yes, and that should be a future enhancement to these tests.  Ultimately we will need to reconcile how to get signing functional.  I was given this URL for when we plan to attempt this: https://github.com/pulp/oci_env/blob/main/profiles/galaxy_base/init.sh